apps/api/drizzle/0030_brief_fixer.sql

```
@@ -0,0 +1,3 @@
+ALTER TABLE "ai_catalog_models" DROP CONSTRAINT "ai_catalog_models_status_check";--> statement-breakpoint
+UPDATE "ai_catalog_models" SET "status" = 'candidate', "updated_at" = now() WHERE "status" = 'retired';--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)